projects/design-system-angular/src/lib/accordion/accordion.component.spec.ts

```diff
@@ -1,25 +1,389 @@
 import { ComponentFixture, TestBed } from '@angular/core/testing';
 
-import { AccordionComponent } from './accordion.component';
+import {
+  ACCORDION_CONTENT_EXPANDED_CLASSNAME,
+  ACCORDION_MULTISELECTABLE_CLASSNAME,
+  AccordionComponent
+} from './accordion.component';
+import {Component, ViewChild} from "@angular/core";
+
+@Component({
+  template: `
+    <jazz-accordion [multiSelect]="false">
```

Comment:
make sure your tests are testing without the multiSelect attribute, we want to make sure the accordions work even without the attribute specified